boards/feather-m0/include/periph_conf.h

```diff
@@ -177,9 +177,11 @@ static const adc_conf_chan_t adc_channels[] = {
     { GPIO_PIN(PA, 4), ADC_INPUTCTRL_MUXPOS_PIN4 },     /* A3 */
     { GPIO_PIN(PA, 5), ADC_INPUTCTRL_MUXPOS_PIN5 },     /* A4 */
     { GPIO_PIN(PB, 2), ADC_INPUTCTRL_MUXPOS_PIN10 },    /* A5 */
+    { GPIO_UNDEF, 0 },                                  /* not defined */
```

Comment:
this doesn't make sense, as the RIOT's mapping does not care about the internal ADC channels, so AIN[7] does not need to be on position 7 in this config array. From RIOT's user perspective, you would simply use RIOT's logical line 6, which is mapped to the CPU's channel AIN[7].

Reply:
Ok. As I said, I wasn't sure about this. Fixed.